addon-test-support/helpers.js

```diff
@@ -7,6 +7,13 @@ const DEFAULT_EVENT_OPTIONS = { canBubble: true, cancelable: true };
 const KEYBOARD_EVENT_TYPES = ['keydown', 'keypress', 'keyup'];
 const MOUSE_EVENT_TYPES = ['click', 'mousedown', 'mouseup', 'dblclick', 'mouseenter', 'mouseleave', 'mousemove', 'mouseout', 'mouseover'];
 
+// Reliable way to get the config?
```

Comment:
@rwjblue is there a way to import the Resolver and lookup the config/environment module? We're not using a container here in the test helpers so can't just lookup config:environement.